### PR TITLE
added stars

### PR DESCRIPTION
**Pull request by Basil-Panasiuk**
- [DEMO LINK](https://Basil-Panasiuk.github.io/layout_stars/)
- [TEST REPORT LINK](https://Basil-Panasiuk.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0">
```
Comment (Elena-brv) [inline src/index.html:12]: stars--0 it's extra class, add also class `stars`
image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+.stars--5 {
+  display: flex;
+  width: 100px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```
Comment (Elena-brv) [inline src/style.css:30]: you can add here background with not active star, and then change it to active with nth-child

**Review by Elena-brv — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+.container {
+  margin: auto;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
```
Comment (Elena-brv) [inline src/style.css:18]:
```suggestion
.stars {
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  display: flex;
+  width: 100px;
```
Comment (Elena-brv) [inline src/style.css:20]: checklist

**Review by Elena-brv — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
+.body {
+  margin: 0;
+  padding: 0;
```
Comment (Denchepornyuk) [inline src/style.css:3]: в body немає дефолтного padding тому не потрібного його обнуляти

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**